### PR TITLE
SWARM-1442: Include licenses deliverables.

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,15 +28,21 @@
   <version>5-SNAPSHOT</version>
   <packaging>war</packaging>
 
-  <name>Simple CRUD Examples</name>
-  <description>WildFly Swarm : REST - CRUD</description>
+  <name>WildFly Swarm Booster - Relational Database Backend Mission</name>
```
Comment (heiko-braun) [inline pom.xml:31]: Has this change been requested somewhere?

File: pom.xml [hunk 2]
```diff
@@ -28,15 +28,21 @@
...
     <version.resteasy>3.0.19.Final</version.resteasy>
     <version.postgresql>9.4.1207</version.postgresql>
-
+    <!-- Licenses - plugins -->
+    <version.wildfly.swarm.fraction.plugin>61</version.wildfly.swarm.fraction.plugin>
+    <version.xml-maven-plugin>1.0.1</version.xml-maven-plugin>
+    <version.license-maven-plugin>1.13</version.license-maven-plugin>
+    <version.wildfly-licenses-plugin>1.0.1</version.wildfly-licenses-plugin>
+    <!-- Licenses - JBoss repo URL used in custom-settings -->
+    <jboss.repo.url>https://repository.jboss.org/nexus/content/groups/public</jboss.repo.url>
```
Comment (heiko-braun) [inline pom.xml:44]: Why do we need this URL and how does this relate to the prod repo?

File: src/main/resources/licenses/licenses-html.xsl [hunk 3]
```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0"?>
```
Comment (heiko-braun) [inline src/main/resources/licenses/licenses-html.xsl:1]: Can't we put all templates into the fraction plugin or any another shared deliverable (or location)?

File: src/main/resources/licenses/licenses-transform.xsl [hunk 4]
```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0"?>
```
Comment (heiko-braun) [inline src/main/resources/licenses/licenses-transform.xsl:1]: Similar question as with the templates: Is there a shared location that prevents us form copying these files all over the place?

**Review by heiko-braun — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (mkouba) [reply pom.xml:31]: No, but I believe it makes sense to align the name with official docs.

File: pom.xml (hunk 2 above)
Comment (mkouba) [reply pom.xml:44]: We use this URL in `custom-settings.xml` to specify the repo where to load the productized artifacts from.

File: src/main/resources/licenses/licenses-html.xsl (hunk 3 above)
Comment (mkouba) [reply src/main/resources/licenses/licenses-html.xsl:1]: It's on the plan and it needs to be discussed.

**Pull request merged**